src/codegen/sys/lib_.rs

```diff
@@ -119,6 +121,43 @@ fn generate_bitfields<W: Write>(w: &mut W, items: &[&library::Bitfield])
     Ok(())
 }
 
+fn generate_constants<W: Write>(w: &mut W, env: &Env, constants: &[library::Constant]) -> Result<()> {
+    try!(writeln!(w, ""));
+    for constant in constants {
+        let (mut comment, mut type_) = match parameter_rust_type(env, constant.typ, library::ParameterDirection::In) {
+            Ok(x) => ("", x),
+            Err(x) => ("//", x),
+        };
+        if env.type_status_sys(&format!("{}.{}", env.config.library_name,
+            constant.name)).ignored() {
+            comment = "//";
+        }
+        let mut value = constant.value.clone();
+        if type_ == "&str" {
+            type_ = "&'static str".into();
+            value = format!("\"{}\"", escape_string(&value));
```

Comment:
You likely can avoid escaping by using the [raw string literal syntax](https://doc.rust-lang.org/reference.html#raw-string-literals).